src/Components/Header/Header.js

```diff
@@ -57,7 +57,10 @@ export default class Header extends Component {
                         <NavLink to="/" className="header__navlink"> Contemporary </NavLink>
                         <MoreStyles />
                         <NavLink to="/" className="header__navlink"> Sale </NavLink>
-                        <NavLink to="/products" className="header__navlink" onClick={() => onFilterProducts([])}> Shop All Rugs </NavLink>
+                        <NavLink to="/products"
+                                 className="header__navlink"
+                                 onClick={() => onFilterProducts([])}
+                                 name="shopAllRugs"> Shop All Rugs </NavLink>
```

Comment:
Name is usually used for other purposes. It's better to not abuse "name" attribute. You can use an existing class name.

Reply:
Done.